main.go

```diff
@@ -33,7 +33,7 @@ func init() {
 	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
 	klog.InitFlags(klogFlags)
 	logtostderr := klogFlags.Lookup("logtostderr")
-	logtostderr.Value.Set("true")
+	logtostderr.Value.Set("true") // nolint:errcheck
```

Comment:
I just ignored this. It's a hacky workaround anyway, don't really care if it throws an error here :shrug: